runtime/stdlib/test_test.go

```diff
@@ -726,7 +726,7 @@ func TestAssertEqual(t *testing.T) {
             fun test() {
                 Test.assertEqual("this string", "this string")
             }
-        `
+		`
```

Comment:
Could you maybe undo this [accidental] change? Having whitespaces is preferred over tabs, given rest of the code snippet also uses whitespaces for indentation (in code blocks)

Reply:
Ah yeah, fixed in https://github.com/onflow/cadence/pull/3014/commits/43b146a6435cf1a78e67e1d001bcf2d96a43c4bf